lib/src/main/kotlin/com/swmansion/starknet/provider/rpc/JsonRpcProvider.kt

```diff
@@ -8,15 +8,29 @@ import com.swmansion.starknet.data.serializers.TransactionReceiptPolymorphicSeri
 import com.swmansion.starknet.data.types.*
 import com.swmansion.starknet.data.types.transactions.*
 import com.swmansion.starknet.provider.Provider
-import com.swmansion.starknet.provider.Request
-import com.swmansion.starknet.service.http.HttpRequest
-import com.swmansion.starknet.service.http.HttpService
-import com.swmansion.starknet.service.http.OkHttpService
+import com.swmansion.starknet.service.http.*
 import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
 import kotlinx.serialization.builtins.ListSerializer
 import kotlinx.serialization.builtins.serializer
 import kotlinx.serialization.json.*
 
+@Serializable
+data class JsonRpcRequest(
```

Comment:
Should this be public?

Reply:
`JsonRpcRequest` was used in other file before, I didn't change its visibility later. Now it's private 👍